ibis/pandas/aggcontext.py

```diff
@@ -464,7 +486,7 @@ def agg(
             # collect implementation) then call apply
             if callable(function):
                 return windowed.apply(
-                    make_applied_function(function, args, kwargs), raw=True
+                    wrap_for_apply(function, args, kwargs), raw=True
```

Comment:
I am curious why are we doing `windowed.apply` instead of `windowed.agg` here?

Reply:
I don't think any of this actually gets hit, and I'm actually not sure about its correctness. I just created #2438 to look into this. For this PR I'll just aim to avoid changing its behavior

Reply:
Ok, sound good to me.